km_api/know_me/tests/models/test_km_user_model.py

```diff
@@ -0,0 +1,11 @@
+from know_me import models
+
+
+def test_create(user_factory):
+    """
+    Test creating a KMUser.
+    """
+    models.KMUser.objects.create(
+       image=None,
```

Comment:
For future reference, since it's a nullable field, you don't even have to give an `image` argument. In this case I actually like it though since we're going to be adding a real image here soon.